docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -374,75 +386,69 @@ Iceberg to Trino type mapping:
     - ``UUID``
   * - ``BINARY``
     - ``VARBINARY``
+  * - ``FIXED (L)``
+    - ``BINARY``
+  * - ``BINARY``
+    - ``BINARY``
   * - ``STRUCT(...)``
     - ``ROW(...)``
   * - ``LIST(e)``
     - ``ARRAY(e)``
-  * - ``MAP(k,v)``
+  * - ``MAP``
     - ``MAP(k,v)``
 
+No other types are supported.
+
 Trino to Iceberg type mapping
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Trino supports creating tables with the following types in Iceberg. The table
-shows the mappings from Trino to Iceberg data types:
-
+The connector maps Trino types to the corresponding Iceberg types following
+this table:
 
 .. list-table:: Trino to Iceberg type mapping
-  :widths: 25, 30, 45
+  :widths: 40, 60
   :header-rows: 1
 
   * - Trino type
     - Iceberg type
-    - Notes
   * - ``BOOLEAN``
     - ``BOOLEAN``
-    -
   * - ``INTEGER``
     - ``INT``
-    -
   * - ``BIGINT``
     - ``LONG``
-    -
   * - ``REAL``
     - ``FLOAT``
-    -
   * - ``DOUBLE``
     - ``DOUBLE``
-    -
   * - ``DECIMAL(p,s)``
     - ``DECIMAL(p,s)``
-    -
   * - ``DATE``
     - ``DATE``
-    -
   * - ``TIME(6)``
     - ``TIME``
-    - Other precisions not supported
   * - ``TIMESTAMP(6)``
     - ``TIMESTAMP``
-    - Other precisions not supported
   * - ``TIMESTAMP(6) WITH TIME ZONE``
     - ``TIMESTAMPTZ``
-    - Other precisions not supported
-  * - ``VARCHAR, VARCHAR(n)``
+  * - ``VARCHAR``
     - ``STRING``
-    -
   * - ``UUID``
     - ``UUID``
-    -
   * - ``VARBINARY``
     - ``BINARY``
-    -
+  * - ``BINARY``
+    - ``FIXED (L)``
+  * - ``BINARY``
+    - ``BINARY``
```

Comment:
Trino doesn't have a Binary type. No Trino types map to the Iceberg `Fixed` type.
```suggestion
```